docroot/themes/custom/yukonca_glider/yukonca_glider.theme

```diff
@@ -52,3 +70,20 @@ function yukonca_glider_preprocess_node(&$variables) {
     }
   }
 }
+
+/**
+ * Implements template_proprocess_page().
+ */
+function yukonca_glider_preprocess_page(&$variables) {
+  if (!empty($variables['node'])) {
+    $bundle = $variables['node']->bundle();
+    if ($bundle == 'landing_page' || $bundle == 'landing_page_level_2') {
+      $node = $variables['node'];
+      $mid = $node->field_svg_upload[0]->getValue()['target_id'];
+      $fid = Media::load($mid)->field_media_image[0]->getValue()['target_id'];
+      $file = File::load($fid);
```

Comment:
Why are you so sure that these sections will have value, media, files?

Reply:
And why can't we use media to render images?
Extracting only images from the field is not the best way, because we won't be able to manage image style for this particular section.

Reply:
Good point.  I've edited the code to return the render array of the media entity.  But since I'm rendering the referenced entity from a page preprocessor, I have to set the image style in the code.  So the image style cannot be changed from the admin panel.  I've set it to use the 'medium (220 x 220)' image style.  The code will be applicable to any node page that has 'field_svg_upload' set.